### PR TITLE
Mount Calibration - WIP

### PR DESCRIPTION
**Pull request by wtgee**
Adding a method to calibrate the mount when it becomes lost

Todo:
- [ ] Use the `say` method
- [x] Show image in PAWS

### CODE REVIEW

File: pocs/observatory.py [hunk 1]
```diff
@@ -537,6 +543,96 @@ def close_dome(self):
...
+            that a slew to a given AltAz will at the very least
+            point somewhere above the horizon.
+
+        Performs the following steps:
+             1. Slews to home.
+             2. Slews to target (default Alt=40° and Az=80°).
+             3. Takes a 30 second image with the primary camera.
+             4. Plates solves image and obtains coords for center of image.
+             5. Displays current separation.
+             6. Sets mount to current coords.
```
Comment (jamessynge) [inline pocs/observatory.py]: Is there a point here where it waits for the user to do something with the mount alignment?

File: bin/pocs_shell [hunk 2]
```diff
@@ -92,6 +92,21 @@ class PocsShell(Cmd):
         i = DriftShell()
         i.cmdloop()
 
+    def do_calibrate_mount(self, *arg):
```
Comment (jamessynge) [inline bin/pocs_shell]: *arg seems unncessary.

File: bin/pocs_shell [hunk 3]
```diff
@@ -92,6 +92,21 @@ class PocsShell(Cmd):
...
+        """Calibrates the mount.
+
+        This should only be run during initial setup of the unit
+        or when the mount has become "lost" and no longer knows where
+        it is pointing.
+
```
Comment (jamessynge) [inline bin/pocs_shell]: Extra blank line.

File: pocs/observatory.py [hunk 4]
```diff
@@ -493,6 +499,96 @@ def close_dome(self):
             self.logger.info('Closed dome')
         return self.dome.close()
 
+    def calibrate_mount(self, target=None, exp_time=30):
+        """Calibrates the mount to a target.
```
Comment (jamessynge) [inline pocs/observatory.py]: What does calibration achieve? Reading the steps I don't get a clue of what the end result is.

File: bin/pocs_shell (hunk 2 above)
Comment (wtgee) [reply bin/pocs_shell]: All of the `do_` commands have that. We don't use it here but I just included it for consistency. Can remove.

File: pocs/observatory.py (hunk 4 above)
Comment (wtgee) [reply pocs/observatory.py]: I'll fill out the docs a little more, thanks.

**Comment by wtgee** [comment]
I'm going to convert this to a utility script rather than an integrated part of the Observatory. Closing for now.

**Pull request closed without merging**